packages/pulpcore/python-box/python-box.spec

```diff
@@ -43,10 +43,15 @@ rm -rf %{pypi_name}.egg-info
 %files -n python3-%{srcname}
 %license LICENSE
 %doc README.rst
+%{python3_sitelib}/__pycache__/box.*
+%{python3_sitelib}/box.py
 %{python3_sitelib}/box
```

Comment:
Is this directory still present if there's a `box.py` file? This looks like an odd package

Reply:
fixed (the dir is there, the py is not)